recipe/bld.bat

```diff
@@ -1,18 +1,19 @@
 echo ON
-cargo build --release
-if errorlevel 1 exit 1
-cargo install --path . --root %LIBRARY_PREFIX%
-if errorlevel 1 exit 1
-
-cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
 
-REM https://github.com/xiph/rav1e
+:: cargo-c is a rust build tool that generates shared library and pkgconfig
+:: it is installed into the build prefix
+:: https://github.com/xiph/rav1e
 cargo install cargo-c
```

Comment:
can we just use the package now?

Reply:
We have to wait for the cargo-c package to be available on all the platforms that you want to support.

Reply:
Or... we're cross compiling, so we just need x86 available?